### PR TITLE
Create LICENSE.md

### PR DESCRIPTION
**Pull request by obi-t4**
# About this PR.
add LICENSE.md
Use Apache License 2.0, same as autoware.

# Ticket
https://tier4.atlassian.net/browse/T4PB-11105

### CODE REVIEW

**Comment by obi-t4** [comment]
pytest is failing now, but https://github.com/tier4/ota-client/pull/133 fixes this issue.

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
```
Comment (Youneko) [inline LICENSE.md:4]: [IMO] *.pyへCopyrightの記述も必要だと思うのですが、そちらはどうしますか？

**Review by Youneko — COMMENTED** [review]
@obi-t4 
コメントを記述しました。
ご確認下さい!

**Review by Youneko — DISMISSED** [review]

**Comment by obi-t4** [comment]
@Youneko 
> pytest is failing now, but https://github.com/tier4/ota-client/pull/133 fixes this issue.

779d9a2acf6c10e6abd7d6e76961fd7dba27d5c7 has been cherry-picked and updated, so please review again :pray:

**Review by Youneko — APPROVED** [review]
LGTM!

**Pull request merged**